rpa-home/modules/ROOT/pages/_partials/automation-credits.adoc

```diff
@@ -0,0 +1,25 @@
+
+// tag::create-usage-report-title[]
+== Create a Usage Report
+// end::create-usage-report-title[]
+
+// tag::create-usage-report[]
+Follow these steps to create a usage report in RPA Manager:
+
+. Open the *Process Operations* module.
+. Click *Usage Summary*.
+. Specify the desired date range and click *Create Usage Summary*.
+. Optionally, you can print or download the report by clicking the *Print* (image:print-icon.png[The Print icon, 5%, 5%])button.
```

Comment:
You can make it shorter:
. Optionally, click *Print* (image:print-icon.png[The Print icon, 5%, 5%]) to print or download the report.